ccdproc/combiner.py

```diff
@@ -775,6 +779,10 @@ def combine(img_list, output_file=None,
                 ccd.mask[x:xend, y:yend] = comb_tile.mask
             if ccd.uncertainty is not None:
                 ccd.uncertainty.array[x:xend, y:yend] = comb_tile.uncertainty.array
+            # Clean up any open files
+            del comb_tile
```

Comment:
Not sure if I really like those `del`s. Are these still necessary for reducing the memory?

Reply:
I think so; can run a profile in a moment to check. The only *essential* one, I think is to delete `tile_combiner`. That will, by design, use roughly the amount of memory the user has set as the cap. If we don't delete it, then on the second pass through the loop we'll (briefly) consume twice that memory on line 761 (or 2 depending on commit) as we construct the new combiner.